Guides/Network_endpoints.md

```diff
@@ -1,23 +1,29 @@
-## Ari Network Endpoints
+# Ari Network Details
+
+## Network endpoints
 
 ```
 CHILDCHAIN_URL: “https://ari.omg.network/”
 
 WATCHER_URL: “https://watcher.ari.omg.network/”
 
-PLASMA_CONTRACT_ADDRESS: “0x44de0ec539b8c4a4b530c78620fe8320167f2f74”
-
 WEB3_PROVIDER_URL: "https://rinkeby.infura.io/"
 (or any IPC Provider)
+```
+
+## Contract information
+
+```
+PLASMA_CONTRACT_ADDRESS: “0x44de0ec539b8c4a4b530c78620fe8320167f2f74”
 
+Current Exit Period: 86400 seconds (1 day)
 ```
 
-### Other Details on Ari
+## Release details
 
 ```
-Current Exit Period: 1 Week
```

Comment:
The exit period could be useful for people building. I'd include it.

Reply:
I moved it into the contract section 👉